src/explore-education-statistics-admin/src/pages/methodology/MethodologiesPage.tsx

```diff
@@ -71,7 +72,9 @@ const MethodologiesPage = () => {
                 {model.liveMethodologies.map(methodology => (
                   <tr className="govuk-table__row" key={methodology.id}>
                     <td className="govuk-table__header">
-                      <Link to={`/methodologies/${methodology.id}`}>
+                      <Link
+                        to={methodologyRoutes[0].generateLink(methodology.id)}
```

Comment:
Export the individual routes so you can do this instead: `summaryRoute.generateLink(methodology.id)`